update_prevalence.py

```diff
@@ -797,8 +803,10 @@ def get_jhu_place(self, jhu_line: JHUCommonFields) -> Place:
         else:
             return self.get_country(jhu_line.Country_Region)
 
-    def get_canada_region_place(self, line: CanadaOpenCovidRegions.RegionInfo) -> Place:
-        return self.get_county(line.health_region, state=line.province_full, country="Canada")
+    def get_canada_region_place(self, health_region: CovidTimelineCanadaRegion, province_or_territory: str) -> Place:
+        return self.get_county(health_region.name_short,
+                               state=province_or_territory,
+                               country="Canada")
```

Comment:
The spelled-out province/territory name isn't provided in this API anymore, but a mapping file is provided, so this method now takes in the full province/territory name.

Reply:
Note that the `pt_names` and `hr_names` parameters allow the PT and HR names to be returned in a variety of formats for both the `timeseries` and `summary` routes.